samples/Catalog/Catalog.android.js

```diff
@@ -103,6 +103,22 @@ var examples = [
       component.props.navigation.navigate("PdfView");
     }
   },
+  {
+    name: "PDF View Component Listeners",
```

Comment:
In https://github.com/PSPDFKit/react-native/pull/102/files#diff-c242db1b34aac23f8a3a7c48ccc0085cR99 we named it "Event Listeners". I don't think we need to mention PDF View Component. So how about "Event Listeners" or just "Listeners"?